Measurements/Tests.qs

```diff
@@ -119,14 +130,14 @@ namespace Quantum.Kata.Measurements {
         // cross-test
         // alpha = 0.0 or PI() => !isQubitOne
         // alpha = PI() / 2.0 => isQubitOne
-        DistinguishTwoStates_OneQubit(StatePrep_IsQubitOne, IsQubitA(PI() / 2.0, _));
+        DistinguishTwoStates_OneQubit(StatePrep_IsQubitOne, IsQubitA(PI() / 2.0, _),"|B⟩","|A⟩");
```

Comment:
For this task, since states A and B can be different in each test case, it would be helpful to give the exact representations of |A⟩ and |B⟩ rather than generic variable names. For example, if the learner has a solution that does rotation in different direction, they might still distinguish |0⟩ and |1⟩ correctly but get stuck on |+⟩ and |-⟩ or the states with unequal superposition.

Reply:
Agreed. Output will look like this now:
'Misclassified' |B⟩=(-sin(1π/10)|0⟩ + cos(1π/10)|1⟩) as |A⟩=(cos(1π/10)|0⟩ + sin(1π/10)|1⟩) in 3 test runs.
'Misclassified' |A⟩=(cos(1π/10)|0⟩ + sin(1π/10)|1⟩) as |B⟩=(-sin(1π/10)|0⟩ + cos(1π/10)|1⟩) in 4 test runs.